src/test/java/org/jitsi/meet/test/util/MeetUtils.java

```diff
@@ -297,56 +297,56 @@ public static boolean isXmppConnected(WebDriver participant)
         return res != null && res.equals(Boolean.TRUE);
     }
 
-    /**
-     * Waits until {@code participant} joins the MUC.
-     * @param participant the participant.
-     * @param timeout the maximum time to wait in seconds.
-     */
-    public static void waitForParticipantToJoinMUC(
-        WebDriver participant, long timeout)
-    {
-        TestUtils.waitForBoolean(
-            participant,
-            IS_MUC_JOINED,
-            timeout);
-    }
-
-    /**
-     * Checks whether a participant is in the MUC.
-     *
-     * @param participant the participant.
-     * @return {@code true} if the specified {@code participant} has joined the
-     * room; otherwise, {@code false}
-     */
-    public static boolean isInMuc(WebDriver participant)
-    {
-        Object res = ((JavascriptExecutor) participant)
-            .executeScript(IS_MUC_JOINED);
-        return res != null && res.equals(Boolean.TRUE);
-    }
-
-    /**
-     * Waits 30 sec for the given participant to enter the ICE 'connected'
-     * state.
-     *
-     * @param participant the participant.
-     */
-    public static void waitForIceConnected(WebDriver participant)
-    {
-        waitForIceConnected(participant, 30);
-    }
-
-    /**
-     * Waits for the given participant to enter the ICE 'connected' state.
-     *
-     * @param participant the participant.
-     * @param timeout timeout in seconds.
-     */
-    public static void waitForIceConnected(WebDriver participant, long timeout)
-    {
-        TestUtils.waitForBoolean(
-            participant, ICE_CONNECTED_CHECK_SCRIPT, timeout);
-    }
+//    /**
```

Comment:
there's a bunch of commented out code in all places, was this removed in next commits ?

Reply:
Yep, there is a commit cleaning those.